From 17474a43f0e0fe830b05e0b4762fb53ac66f5707 Mon Sep 17 00:00:00 2001
From: Li Jingyu <b41874@freescale.com>
Date: Wed, 25 Nov 2015 16:05:23 +0800
Subject: [PATCH] remove redefined macros in common_config.h to support new
 u-boot

Following macros have been defined default in u-boot 2014.xx and later versions.
CONFIG_CMD_NET
CONFIG_CMD_MISC
CONFIG_CMD_RUN
CONFIG_CMD_SAVEENV
CONFIG_CMD_SETEXPR
CONFIG_CMD_SOURCE
CONFIG_CMD_XIMG
CONFIG_SYS_CBSIZE
CONFIG_CMD_MEMORY
CONFIG_CMD_CONSOLE
CONFIG_CMD_FLASH
CONFIG_CMD_IMI
CONFIG_CMD_BDI
CONFIG_CMD_EDITENV

Signed-off-by: Li Jingyu <b41874@freescale.com>
---
 include/configs/common_config.h | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/include/configs/common_config.h b/include/configs/common_config.h
index e9e2135..f406772 100644
--- a/include/configs/common_config.h
+++ b/include/configs/common_config.h
@@ -36,35 +36,19 @@
  *
  */
 #define CONFIG_CMD_ASKENV	/* ask for env variable		*/
-#define CONFIG_CMD_BDI		/* bdinfo			*/
-#define CONFIG_CMD_BOOTD	/* bootd			*/
-#define CONFIG_CMD_CONSOLE	/* coninfo			*/
 #undef	CONFIG_CMD_DATE		/* support for RTC, date/time...*/
 #define CONFIG_CMD_DHCP		/* DHCP Support			*/
 #define CONFIG_CMD_DIAG		/* Diagnostics			*/
-#define CONFIG_CMD_ECHO		/* echo arguments		*/
-#define CONFIG_CMD_EDITENV	/* editenv			*/
 #define CONFIG_CMD_EXT2		/* EXT2 Support			*/
 #define CONFIG_CMD_FAT		/* FAT support			*/
-#define CONFIG_CMD_FLASH	/* flinfo, erase, protect	*/
 #define CONFIG_CMD_I2C		/* I2C serial bus support	*/
-#define CONFIG_CMD_IMI		/* iminfo			*/
 #define CONFIG_CMD_IRQ		/* irqinfo			*/
-#define CONFIG_CMD_ITEST	/* Integer (and string) test	*/
 #define CONFIG_CMD_JFFS2	/* JFFS2 Support		*/
-#define CONFIG_CMD_MEMORY	/* md mm nm mw cp cmp crc base loop mtest */
 #define CONFIG_CMD_MII		/* MII support			*/
-#define CONFIG_CMD_MISC		/* Misc functions like sleep etc*/
-#define CONFIG_CMD_NET		/* bootp, tftpboot, rarpboot	*/
 #define CONFIG_CMD_PCI		/* pciinfo			*/
 #define CONFIG_CMD_PING		/* ping support			*/
-#define CONFIG_CMD_RUN		/* run command in env variable	*/
-#define CONFIG_CMD_SAVEENV	/* saveenv			*/
-#define CONFIG_CMD_SETEXPR	/* setexpr support		*/
 #define CONFIG_CMD_SETGETDCR	/* DCR support on 4xx		*/
 #define CONFIG_CMD_SDRAM	/* SDRAM DIMM SPD info printout */
-#define CONFIG_CMD_SOURCE	/* "source" command support	*/
-#define CONFIG_CMD_XIMG		/* Load part of Multi Image	*/
 
 /*
  * Configuration options common to onie installation platforms.
@@ -75,7 +59,6 @@
 #define CONFIG_CMDLINE_EDITING		/* Command-line editing */
 #define CONFIG_AUTO_COMPLETE		/* add autocompletion support */
 
-#define CONFIG_SYS_CBSIZE	2048	/* Console I/O Buffer Size */
 /* Print Buffer Size */
 #define CONFIG_SYS_PBSIZE (CONFIG_SYS_CBSIZE+sizeof(CONFIG_SYS_PROMPT)+16)
 #define CONFIG_SYS_MAXARGS	16		/* max number of command args */
-- 
1.9.1

